### ISSUE TITLE
Old Aperture Chamberlock needs a styled editor model

### ISSUE THREAD

**Issue by LautaroL20** [BEEmod/BEE2-items#704]
Title says it all.

**Comment by ZombiePwner1** [comment]
LOL i'm not missing the PeTi model.

**Comment by LautaroL20** [comment]
@ZombiePwner1 - Thats probably because TeamSpen210 gave you some files. As i always say, i make sure to install everything correct before reporting any bugs.

**Comment by ZombiePwner1** [comment]
He didn't give me any files.

**Comment by ZombiePwner1** [comment]
BTW come online :)

**Comment by LautaroL20** [comment]
Ah, your Chamberlock model is working because you have the original addons pack installed. I installed bee2.4 on a 100% clean install from P2.

**Comment by ZombiePwner1** [comment]
:D

**Comment by ZombiePwner1** [comment]
@LautaroL20 the reason why it's missing an editor model is because one hasn't been made yet.

**Comment by Luke18033** [comment]
Fixed now?

**Comment by LautaroL20** [comment]
@Luke18033 - Yes.

**Comment by TeamSpen210** [comment]
Not quite, it's a new editor model, but we'd want a Old Aperture one too since they look pretty different.
